jobspec/test-fixtures/tg-service-connect-gateway-ingress.hcl

```diff
@@ -24,6 +24,8 @@ job "connect_gateway_ingress" {
           ingress {
             tls {
               enabled = true
+              tls_min_version = "TLSv1_2"
+              cipher_suites = ["TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256"]
```

Comment:
I think you will need to `hclfmt` this file. `make hclfmt` should take care of it.